src/cc/tools/qfsfileenum_main.cc

```diff
@@ -43,16 +43,18 @@ using std::setw;
 int
 main(int argc, char **argv)
 {
-    bool        help           = false;
-    const char* server         = 0;
-    const char* filename       = 0;
-    const char* config         = 0;
-    int         port           = -1;
-    bool        verboseLogging = false;
-    int         retval;
+    bool        help               = false;
```

Comment:
why do we do this indentation like this? It makes the addition of a few variables change the entire block...